packages/melos/lib/src/command_runner.dart

```diff
@@ -55,4 +63,14 @@ class MelosCommandRunner extends CommandRunner<void> {
     addCommand(PublishCommand());
     addCommand(VersionCommand());
   }
+
+  @override
+  Future<void> runCommand(ArgResults topLevelResults) async {
+    if (topLevelResults['version'] == true) {
+      // ignore: avoid_print
+      print(melosVersion);
```

Comment:
This probably should use the logger

Reply:
logger is a instance property of `MelosCommand` whereas this isn't a command instance 🤔

Reply:
Leaving as is for now as discussed on slack